doc/source/ray-observability/state/state-api.rst

```diff
@@ -251,26 +251,26 @@ E.g., Get a particular log file from a node
 .. code-block:: bash
 
     # You could get the node id / node ip from `ray list nodes` 
-    ray logs gcs_server.out --node-id <XYZ> 
+    ray logs file gcs_server.out --node-id <XYZ> 
 
 E.g., Stream a log file from a node
 
 .. code-block:: bash
 
-    ray logs -f raylet.out --node-ip 172.31.47.143
+    ray logs file -f raylet.out --node-ip 172.31.47.143
 
 E.g., Stream actor log with actor id 
 
 .. code-block:: bash
 
     # You could use ray list actors to get the actor ids
```

Comment:
Can you change the order of examples actually? We should bring up high level API before the low level API (so ray logs actor & ray logs worker is more important than ray logs file)